### PR TITLE
Fixed address & balance summary.

### LINKED ISSUE #1: Fix cennznet-ui
- Update cennznet-ui with centrality logo
- Update balance to support GA dual token: e.g. staking tab, account tab, address book, extrinsic
- update github tab to point to cennznet-ui repo
- update wiki tab to point to cennznet-doc
...

### PR DESCRIPTION
**Pull request by shaunxw**
By default address summary would show spending tokens(centrapay) balance, in some cases like staking tab, staking token(cennz) balance would be shown.

Fixes #1

### CODE REVIEW

File: packages/ui-app/src/AddressSummary.tsx [hunk 1]
```diff
@@ -137,10 +139,19 @@ class AddressSummary extends React.PureComponent<Props> {
   }
 
   protected renderBalance () {
-    const { accounts_idAndIndex = [], balance, t, value, withBalance = true } = this.props;
+    const { accounts_idAndIndex = [], balance, t, value, withBalance = true, assetId = 'CENTRAPAY-T' } = this.props;
     const [_accountId] = accounts_idAndIndex;
     const accountId = _accountId || value;
 
+    let label;
+    if (assetId === 'CENTRAPAY-T') {
```
Comment (xlc) [inline packages/ui-app/src/AddressSummary.tsx]: it is ok to display `CENTRAPAY-T` or `CENNZ-T`. good to inform user they are test token.

**Review by xlc — CHANGES_REQUESTED** [review]
Just one small change

File: packages/ui-app/src/AddressSummary.tsx (hunk 1 above)
Comment (shaunxw) [reply packages/ui-app/src/AddressSummary.tsx]: Updated.

**Review by xlc — APPROVED** [review]

**Comment by xlc** [comment]
Needs fix build issue

**Comment by shaunxw** [comment]
The build failure is caused by some legacy typescript compiling errors non-related to this PR.
Will fix them anyway.

**Pull request merged**